panimg/image_builders/dicom.py

```diff
@@ -215,11 +215,14 @@ def _process_dicom_file(*, dicom_ds):  # noqa: C901
                 origin_diff = origin_diff + diff
                 n_diffs += 1
             origin = file_origin
-    avg_origin_diff = tuple(origin_diff / n_diffs)
-    try:
-        z_i = avg_origin_diff[2]
-    except IndexError:
-        z_i = 1.0
+    if n_diffs == 0:
+        z_i = np.nan
+    else:
+        avg_origin_diff = tuple(origin_diff / n_diffs)
```

Comment:
Does this not always result in a tuple of length 1, since both `origin_diff` and `n_diffs` are ints now?

Reply:
Which probably also means we need more test cases to cover this, as tests are now passing.

Reply:
Ah that makes sense, thank you for clarifying.